pkg/tuf/common.go

```diff
@@ -1,24 +1,19 @@
-// Most of the helper functions are adapted from github.com/theupdateframework/notary
-//
-// Figure out the proper way of making sure we are respecting the licensing from Notary
-// While we are also vendoring Notary directly (see LICENSE in vendor/github.com/theupdateframework/notary/LICENSE),
-// copying unexported functions could fall under different licensing, so we need to make sure.
-
 package tuf
```

Comment:
Are there still functions that were copied from TUF into this repo? If so, are they separated into their own files with a copyright/license header, linking to the original source file and noting any modifications?